test/parallel/test-trace-events-fs-sync.js

```diff
@@ -6,7 +6,7 @@ const fs = require('fs');
 const path = require('path');
 const util = require('util');
 
-const tests = new Array();
+const tests = {};
```

Comment:
IIRC using null-prototype object helps V8 optimize the object for dictionary usage.
```suggestion
const tests = Object.create(null);
```

Reply:
Thanks. updated.